python/ctsm/test/test_unit_neon_surf_wrapper.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+"""
+Unit tests for neon_surf_wrapper
+
+You can run this by:
+    python -m unittest test_unit_neon_surf_wrapper.py
+"""
+
+import unittest
+import configparser
+import argparse
+import os
+import sys
+
+# -- add python/ctsm  to path (needed if we want to run the test stand-alone)
+_CTSM_PYTHON = os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir)
+sys.path.insert(1, _CTSM_PYTHON)
+
+# pylint: disable=wrong-import-position
+from ctsm import unit_testing
+from ctsm.site_and_regional.neon_surf_wrapper import get_parser
+from ctsm.path_utils import path_to_ctsm_root
+
+# pylint: disable=invalid-name
+
+
+class TestNeonSurfWrapper(unittest.TestCase):
+    """
+    Basic class for testing neon_surf_wrapper.py.
+    """
+
+    def test_parser(self):
+        """
+        Test that parser has same defaults as expected
+        """
+
+        self.assertEqual(get_parser().argument_default, None, "Parser not working as expected")
```

Comment:
This one is new, right? 
If I'm understanding correctly, if blank, then fail. Seems good.

Reply:
This is actually saying that there should be no default argument, as the argument_default should be equal to None. If some default argument were added, we would want to be aware of how that would affect `run_neon` when the execute command is called. I could also change it to assert some other part of the parser is working as expected.

Reply:
Oh, I misunderstood, thank you for explaining. I think it's fine to leave this way for now.